src/ngMock/angular-mocks.js

```diff
@@ -1346,11 +1346,15 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
                  copy(response[3] || ''));
       }
 
-      function handleTimeout() {
+      function handleTimeout(statusText) {
         for (var i = 0, ii = responses.length; i < ii; i++) {
           if (responses[i] === handleResponse) {
             responses.splice(i, 1);
-            callback(-1, undefined, '');
+            if (statusText) {
```

Comment:
Because some unit tests use spy's ```toHaveBeenCalledWith()``` method which is strict regarding the number of parameters, and I didn't want to break the old API. With ```undefined``` as the forth parameter those tests will fail, so may be some third-party modules would break if they test for it. It's minor however, and if you agree, we can just change it.